.github/workflows/eden.yml

```diff
@@ -34,10 +34,10 @@ jobs:
       - name: run
         run: |
           ./eden config add default
-          ./eden config set default --key eve.tag --value=$(make -C eve version)
+          ./eden config set default --key eve.tag --value=$(make -s -C eve version)
```

Comment:
Adding -s if fine, but I'm wondering why do we need it. Running:
```
make -C eve version
0.0.0-master-e8a42616
```
on my laptop doesn't produce anything but the version ID. What am I missing?

Reply:
Answering my own question ;-) it seems to be a difference between macOS BSD make and Linux GNU make -- so yeah your `-s` is completely justified. Lets just fix the thing below and we can merge this.